__tests__/app.test.js

```diff
@@ -221,7 +220,257 @@ describe("POST /api/articles/:article_id/comments", () => {
+      .send({ inc_votes: 25 })
+      .expect(400)
+      .then(({ res }) => {
+        expect(res.statusMessage).toBe('Invalid request: not a valid integer entered!');
+      });
+  });
+  test("PATCH 400: responds with 'Invalid request...' when the votes parameter is in the wrong format", () => {
+    return request(app)
+      .patch("/api/articles/2")
+      .send({ inc_votes: "twenty-five" })
+      .expect(400)
+      .then(({ res }) => {
+        expect(res.statusMessage).toBe("Invalid request: not a valid integer entered!");
+      });
+  });
+  test("PATCH 400: responds with 'Invalid request...' when the inc_votes property is missing", () => {
+    return request(app)
+      .patch("/api/articles/2")
+      .send({})
+      .expect(400)
+      .then(({ res }) => {
+        expect(res.statusMessage).toBe("Invalid request: not a valid integer entered!");
+      });
+  });
+})
+
+describe("GET /api/articles/:article_id?comment_count=true", () => {
+  test("GET 200: responds with article object, which should have comment count property when comment_count=true", () => {
+    return request(app)
+      .get("/api/articles/2?comment_count=true")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.article).toEqual({
+          article_id: 2,
+          title: 'Sony Vaio; or, The Laptop',
+          topic: 'mitch',
+          author: 'icellusedkars',
+          body: expect.any(String),
+          created_at: "2020-10-16T05:03:00.000Z",
+          votes: 0,
+          article_img_url: expect.any(String),
+          comment_count: expect.any(Number),
+        });
+        expect(body.article.comment_count).toBeGreaterThanOrEqual(0);
+      });
+  });
+  test("GET 200: responds with article object, which should not have comment count property when comment_count=false or not provided", () => {
+    return request(app)
+      .get("/api/articles/2?comment_count=false")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.article).toEqual({
+          article_id: 2,
+          title: 'Sony Vaio; or, The Laptop',
+          topic: 'mitch',
+          author: 'icellusedkars',
+          body: expect.any(String),
+          created_at: "2020-10-16T05:03:00.000Z",
+          votes: 0,
+          article_img_url: expect.any(String),
+        });
+        expect(body.article.comment_count).toBeUndefined();
+      })
+      .then(() => {
+        return request(app)
+          .get("/api/articles/2")
+          .expect(200);
+      })
+      .then(({ body }) => {
+        expect(body.article).toEqual({
+          article_id: 2,
+          title: 'Sony Vaio; or, The Laptop',
+          topic: 'mitch',
+          author: 'icellusedkars',
+          body: expect.any(String),
+          created_at: "2020-10-16T05:03:00.000Z",
+          votes: 0,
+          article_img_url: expect.any(String),
+        });
+        expect(body.article.comment_count).toBeUndefined();
+      });
+  });
+});
```

Comment:
You have misinterpreted this ticket. Comment count should now be applied to ALL get requests on the path of api/articles/2 etc. It does _not_ need a query parameter. 

You should just expand the original functionality to include a comment count and write a separate test